src/app.ts

```diff
@@ -46,6 +52,7 @@ class App {
   }
 
   private config(): void {
+    this.app.use('/auth/requestResetPassword', pwRequestLimiter);
```

Comment:
This is rather misplaced. I know the example in the repo says to do it like this, but it should also work in the `routes` file, just add it as a middleware there. See how `authMiddleware` is used there. But please test if it works as expected.
And FYI you can add as many middlewares as you like for any given route, e.g. `route(...).get(middleware1, middleware2, middleware3, handler)`. This is not relevant right now but just so you know